Makefile

```diff
@@ -41,7 +40,7 @@ test-unit-integration:
 check-all: validate-models-yml check-models check-initial-data-json check-example-data-json check-permissions
 
 validate-models-yml:
-	python global/meta/models_validator/validate.py global/meta/models.yml
+	make -C global/meta/dev validate-models
```

Comment:
IMO we should remove this test in backend. The integrity of model.yml should be the concern of the meta repo

Reply:
We should keep it in the Makefile as an easy-to-access way of executing the check (otherwise you have to manually go into the `global/meta` folder). But if you want, we can remove it from the CI. I agree that it's redundant.

Reply:
I agree with the easy-to-access way of checking from backend during dev.
What do you think from removing the models validation from .github/workflows/continuous_integration.yml? That would be sufficient for me

Reply:
done